client/js/verification-portal/community-health-center/action-creators.js

```diff
@@ -0,0 +1,23 @@
+import { change } from 'redux-form'
+
+import { store } from './../../hah-app/index'
+import { chcInputResource } from './chc-input-resource'
+
+const clearField = inputField => {
+	if (!inputField.name) return inputField.forEach(field => clearField(field))
+	store.dispatch(change('chcForm', inputField.verify, ''))
+	return store.dispatch(change('chcForm', inputField.name, ''))
+}
+
+const submit = values => {
+	return {
+		type: 'SUBMIT_CHC_FORM',
```

Comment:
Do you mind putting the types in an `action-types` file? Also we should be prepending them with a unique word to categorize the types of actions. Here is an example: https://github.com/HelpAssistHer/help-assist-her/blob/master/client/js/verification-portal/pregnancy-resource-center/action-types.js